manifests/server/extension.pp

```diff
@@ -1,6 +1,7 @@
 # Activate an extension on a postgresql database
 define postgresql::server::extension (
   $database,
+  $extension = $name,
```

Comment:
Discussed as it is a database level extension and might be needed for different databases.